# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/webrender_build/src/shader_features.rs
# Commit: 0edb6b6f17b2
# Full Hash: 0edb6b6f17b24c737d053df19730e9c9e61e17ca
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2020-12-18 04:07:40
# Regressor Bug: 1682365
# File Overlap Count: 1
# Description:
#   Bug 1682365 - Pt 2 - Remove texture array usage from render task graph. r=nical
#   
#   With this change, all color/alpha intermediate surfaces are individual
#   textures, rather than texture arrays.
#   
# ==============================================================================

diff -r ee0de0eac18d -r 0edb6b6f17b2 gfx/wr/webrender_build/src/shader_features.rs
--- a/gfx/wr/webrender_build/src/shader_features.rs	Thu Dec 17 22:26:08 2020 +0000
+++ b/gfx/wr/webrender_build/src/shader_features.rs	Thu Dec 17 22:26:10 2020 +0000
@@ -109,7 +109,7 @@
     }
 
     // Image brush shaders
-    let mut texture_types = vec!["", "TEXTURE_2D"];
+    let mut texture_types = vec!["TEXTURE_2D_ARRAY", "TEXTURE_2D"];
     if flags.contains(ShaderFeatureFlags::GL) {
         texture_types.push("TEXTURE_RECT");
     }